README.md

```diff
@@ -130,3 +130,62 @@ To debug native code with VSCode or CLion or any other IDE:
 3. Set the parameters to be the ones used by the ```mvn``` script, as per above
 4. Set the working directory to the `aws-crt-java` directory
 5. On windows, you will need to manually load the PDB via the Modules window in Visual Studio, as it is not embedded in the JAR. It will be in the ```target/cmake-build/lib/windows/<arch>``` folder.
+
+## Platform-Specific JARs
+
+Since v0.20.5, platform-specific JARs are available to reduce the size of the artificts. To consume the platform-specific JAR of aws-crt-java, you can use Maven classifier (https://maven.apache.org/pom.html) property to specify the platform. Without classifier, you will get the uber jar to support all distributions.
+
+``` xml
+        <!-- Linux x86_64 distribution  -->
+        <dependency>
+            <groupId>software.amazon.awssdk.crt</groupId>
+            <artifactId>aws-crt</artifactId>
+            <version>0.20.5</version>
+            <classifier>linux-x86_64</classifier>
+        </dependency>
+
+        <!-- Uber JAR with all distributions -->
+        <dependency>
+            <groupId>software.amazon.awssdk.crt</groupId>
+            <artifactId>aws-crt</artifactId>
+            <version>0.20.5</version>
+        </dependency>
+```
+
+### Available classifier
+- linux-armv6
+- linux-armv7
+- linux-aarch_64
+- linux-x86_32
+- linux-x86_64
+- osx-aarch_64
+- osx-x86_64
+- windows-x86_32
+- windows-x86_64
+
+### Auto-detected
+There are build plugins such as os-maven-plugin (https://github.com/trustin/os-maven-plugin) that help project automatically determine the OS at build time.
+
+**NOTES**: For auto-detected `linux-arm_32` archtecture, you need to specific `linux-armv6` or `linux-armv7` to consume the excat platform-specific jar.
```

Comment:
minor: i would reword it slightly to say that we follow common platform names (as in the link) to make it easier to detect platform. But for arm_32 we have 2 types that are not compatible and user needs to manually make the decision.

Reply:
are we sure that aarch_64 build are compatible across all arm 64 types? arm 8/9/etc...?

Reply:
TBH, I don't know, but I think if we don't support all the arm64 other than armv8, we should know, as it is not supported now as well. This change doesn't break/change the behavior for those type.